_aggregations/pipeline/moving-avg.md

```diff
@@ -0,0 +1,568 @@
+---
+layout: default
+title: Moving average
+parent: Pipeline aggregations
+nav_order: 120
+redirect_from:
+  - /query-dsl/aggregations/pipeline-agg#moving_avg/
+  - /query-dsl/aggregations/pipeline/moving-avg/
+---
+
+## Moving average aggregations
+
+A `moving_avg` aggregation is a parent pipeline aggregation that calculates a sequence of averages of a metric contained in windows (adjacent subsets) of an ordered dataset. 
```

Comment:
From code:

"The moving_avg aggregation has been deprecated in favor of the moving_fn aggregation."

---------

Since this seems to be on deprecation path, I think it might be better to document `moving_fn` instead.

Reply:
If the functionality is still in the product, it should be documented, even if it's deprecated.

Deprecation notices should be included in the release notes. It can also be mentioned in the doc for the deprecated functionality if that's documentation policy. 

@kolchfa-aws , should this page contain a deprecation notice?

I'm in the process of also documenting the moving_fn aggregation; a PR should be available soon.

Reply:
@dwelsch-esi Yes, there should be a deprecation notice for this one, advising to use moving_fn aggregation instead.

Reply:
Added a deprecation notice.